webapp/packages/plugin-object-viewer/src/ObjectPropertiesPage/ObjectPropertyTable/ObjectPropertyTableFooter.tsx

```diff
@@ -20,20 +22,10 @@ interface Props {
 }
 
 export const ObjectPropertyTableFooter = observer<Props>(function ObjectPropertyTableFooter({ nodeIds, tableState, className }) {
-  const service = useService(ObjectPropertyTableFooterService);
+  const menu = useMenu({ menu: MENU_OBJECT_VIEWER_FOOTER_ACTIONS });
 
-  const items = service.constructMenuWithContext(nodeIds, tableState);
-  const hidden = items.every(item => item.isHidden);
+  menu.context.set(DATA_CONTEXT_OBJECT_VIEWER_NODES, nodeIds);
+  menu.context.set(DATA_CONTEXT_OBJECT_VIEWER_TABLE_STATE, tableState);
```

Comment:
i will suggest to use `DATA_CONTEXT_NAV_NODE` and `DATA_CONTEXT_NAV_NODES`
https://github.com/dbeaver/cloudbeaver/blob/f8a36c09ee11eba183b852cf70832f5b1f696ecf/webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/NavigationTreeNode/NavigationNode.tsx#L67-L69